src/graph/entities/attribute_set.h

```diff
@@ -25,7 +25,7 @@ typedef struct {
 } Attribute;
 
 typedef struct {
-	ushort attr_count;       // number of attributes
+	uint16_t attr_count;       // number of attributes
```

Comment:
Why changing from `ushort` to `uint16_t` I guess we want to be explicit regarding the size of the type ?